### ISSUE TITLE
Design not completely fluid and no option to add styles to components

### ISSUE THREAD

**Issue by imixtron** [ixartz/Astro-boilerplate#7]
1. Adding extra Categories Tags breaks the UI fluidity on mobile apps.
2. Hero banner Avatar looks fine with provided image but its breaking other images that do not have transparent padding by default.

Also components do not take any style props or it could have been fixed by adding the right css style.

<img width="415" alt="image" src="https://user-images.githubusercontent.com/10543720/185488323-bcc39a2e-1268-4e8c-9b42-40c83512e9ea.png">

<img width="413" alt="image" src="https://user-images.githubusercontent.com/10543720/185488503-70b25ae4-2305-473e-b6b5-6db84d31fb05.png">

**Comment by ixartz** [comment]
@imixtron, the component is inside another repository: https://github.com/ixartz/astro-boilerplate-components

It was a requirement from Astro hackathon to have a separate repository. Totally open for pull request.

1. This is totally fixable with Tailwind CSS.
2. You don't have much choice to use transparent background. This isn't related to the template. When the background is dark, you need either a similar color or transparent background. You have a lot of online tool which can make the background transparent in your image.

**Comment by yoriscape** [comment]
You can configure position of buttons you're placing.

**Comment by ShahRishi** [comment]
> @imixtron, the component is inside another repository: https://github.com/ixartz/astro-boilerplate-components
> 
> It was a requirement from Astro hackathon to have a separate repository. Totally open for pull request.
> 
> 1. This is totally fixable with Tailwind CSS.
> 2. You don't have much choice to use transparent background. This isn't related to the template. When the background is dark, you need either a similar color or transparent background. You have a lot of online tool which can make the background transparent in your image.

Regarding this, is there a way we can edit the astro-boilerplate-components for our own projects? Thanks!

**Comment by ixartz** [comment]
Yes, you can edit for your own project and the repository is here: https://github.com/ixartz/astro-boilerplate-components

**Comment by ShahRishi** [comment]
Sorry but I wanted to clarify, do how do I merge this repo with the template?

**Comment by ixartz** [comment]
If you want to merge, you can copy the files from the other repository.
